Neos.Fusion/Classes/Core/Parser.php

```diff
@@ -512,11 +512,13 @@ protected function parseValueAssignment($objectPath, $value)
      *
      * @param string $objectPath The object path as a string
      * @return void
+     * @throws Fusion\Exception
      */
     protected function parseValueUnAssignment($objectPath)
     {
         $objectPathArray = $this->getParsedObjectPath($objectPath);
         $this->setValueInObjectTree($objectPathArray, null);
+        $this->setValueInObjectTree($objectPathArray, ['__valueUnAssignment' => true]);
```

Comment:
The key `__valueUnAssignment ` is a bit confusing. How about a more explicit name like `__stopInheritenceChain`